STYLE_API_TEMPLATE.md

```diff
@@ -52,6 +53,14 @@ If applicable, provide any caveats to its usage with a note or tip, as in the fo
 "If you use the Security plugin, make sure you have the appropriate permissions."
 (To set this point in note-style format, follow the text on the next line with {: .note})
 
+### Path and HTTP methods
```

Comment:
```suggestion
```

Reply:
Need to move paragraph and description to the proper section. This PUT example call goes in the Request fields section with the other PUT/POST API examples that have requests fields.